src/ui/component/skeleton/skeleton.tsx

```diff
@@ -0,0 +1,52 @@
+import type { CSSProperties, FC } from 'react'
+import classNames from 'classnames'
+import './skeleton.scss'
+
+type SkeletonProps = {
+  width?: number | string
+  height?: number | string
+  fontSize?: number | string
+  variant?: 'text' | 'rectangular' | 'rounded' | 'circular'
+  className?: string
+  style?: CSSProperties
+  count?: number
+}
+
+export const Skeleton: FC<SkeletonProps> = ({
+  width,
+  height,
+  variant = 'text',
+  fontSize,
+  className,
+  style,
+  count = 1
+}) => {
+  const isRounded = variant === 'rounded'
+
+  if (isRounded && !height) {
+    throw new Error(
+      "A height must be provided when using the 'rounded' variant of Skeleton component."
+    )
+  }
```

Comment:
Great to throw an error, but who will catch it ?

Reply:
I'll suggest to remove the error for now, the `ErrorBoundary` will be added in a future iteration with the 500 page feature.